core/services/log/mocks/orm.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery 2.9.0. DO NOT EDIT.
+// Code generated by mockery v2.8.0. DO NOT EDIT.
```

Comment:
Uh oh, you went backwards here

Reply:
Ugh, can we disable these version headers? `--disable-version-string`